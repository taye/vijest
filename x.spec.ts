test('snapshots', () => {
  expect(1).toMatchSnapshot()

  jest.useFakeTimers()

  const callback = jest.fn(() => {
    vt.debugger()
  })

  setTimeout(callback, 4000)
  expect(callback).not.toHaveBeenCalled()
  jest.runAllTimers()

  expect(callback).toHaveBeenCalled()

  expect(100).toMatchInlineSnapshot('100')
  // expect([100]).toMatchInlineSnapshot('Array [\n   100,\n]')
})
