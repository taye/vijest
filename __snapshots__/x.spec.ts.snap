// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`X 1`] = `1`;

exports[`snapshots 1`] = `1`;
